sw/airborne/state.h

```diff
@@ -691,6 +701,15 @@ static inline struct UtmCoor_f *stateGetPositionUtm_f(void)
   return &state.utm_pos_f;
 }
 
+/// Get altitude above ground level (float).
+static inline float *stateGetAltAgl_f(void)
```

Comment:
Should directly return a float instead of a pointer to it (like the other functions returning a primitive type).